auto-self-nominate/src/index.ts

```diff
@@ -137,14 +129,26 @@ function supplementaryDeposit(
     if (!info.hasOwnProperty(address)) {
         return targetDeposit;
     }
-    const [deposit] = info[address];
-    if (deposit < targetDeposit) {
-        return targetDeposit - deposit;
+    const { deposits } = info[address];
+    if (deposits < targetDeposit) {
+        return targetDeposit - deposits;
     } else {
         return 0;
     }
 }
 
+function sendAlarmForBanned(address: string) {
+    const content = `Your account ${address} is permanently banned`;
```

Comment:
It would be better to show the reason why the account is banned.
Could you create an issue about it?

Reply:
Do you mean the specific reason including the information about in what voting step the double voting was detected?

Reply:
Yes. The two votes which are evidence of the DoubleVote transaction.

Reply:
Ok, I created issue related it.